.github/ISSUE_TEMPLATE/stable_feature.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Stable Feature Requirements
+about: Requirements for promoting a feature to stable
+---
+
+This page lists the requirements for promoting a feature to stable. Please check off and document the steps as they are completed.
+
+**Feature Name:** 
+--- 
+
+### Requirements: 
+
+**Tests**
+
+- [ ] Test coverage is >= 90%
+- [ ] Automated tests are in place to prevent performance regressions. 
+
+**Performance**
+
+- [ ] Latency, throughput, and scale are quantified and documented. 
+
+**Bugs**
+
+- [ ] Feature has no outstanding P0 or P1 bugs. 
+
+**Performance**
```

Comment:
```suggestion
**Promotion**
```

Reply:
Fixed. Thanks.

Reply:
Doesn't look fixed.

I think we should actually call this section Approvals or Reviews though.